packages/documentation/src/stories/components/button/button.docs.mdx

```diff
@@ -59,3 +59,10 @@ For example, a button with classes `w-100 w-lg-auto` will use all available spac
 To learn more about breakpoints, see the <a href="/?path=/docs/foundations-layout-breakpoints--docs">Breakpoints documentation</a>.
 
 <Canvas of={ButtonStories.FullWidth} />
+
+### Align buttons
+
+Use flexbox to group and align buttons. As a best practice for keyboard navigation, order the most important button as the first occurrence in a group in the HTML
+and use the `flex-row-reverse`, `flex-column-reverse` classes or the `order` CSS property to change the visual order. In the following example, the send button appears first on tab navigation.
```

Comment:
```suggestion
and use the `.flex-row-reverse`, `.flex-column-reverse`, or `.order-*` classes to change the visual order.

In the following example, the send button appears first on tab navigation.
```

Since they exist: https://getbootstrap.com/docs/5.3/utilities/flex/#order
It would be good to document them, don't you think? If so, we can open a new ticket.